docs/src/pages/demos/popovers/popovers.md

```diff
@@ -14,7 +14,7 @@ When it is `anchorPosition`, the component will, instead of `anchorEl`,
 refer to the `anchorPosition` prop which you can adjust to set
 the position of the popover.
 
-{{"demo": "pages/demos/popovers/AnchorPlayground.js"}}
+{{"demo": "pages/demos/popovers/AnchorPlayground.js", "hideEditButton": true}}
```

Comment:
What's wrong with this one?

Reply:
I get an HTTP error 414. Can't figure out why.

Reply:
Should i leave the link in place for this one?